tests/cpu.rs

```diff
@@ -46,4 +49,5 @@ fn test_global_cpu_info_not_set() {
     assert_eq!(s.global_cpu_info().vendor_id(), "");
     assert_eq!(s.global_cpu_info().brand(), "");
     assert_eq!(s.global_cpu_info().frequency(), 0);
+    assert!(s.global_cpu_info().arch() == sysinfo::CpuArch::UNKNOWN);
```

Comment:
It definitely make more sense to store `Option<CpuArch>` now that I think about `global_cpu` since it's not supposed to have this information.